Widgets/Bars/ClassBar.lua

```diff
@@ -641,6 +671,12 @@ function W:CreateClassBar(button)
         end
     end
 
+    function classBar:ShowBars()
```

Comment:
This function should be deleted, `UpdateSize` deals with showing the proper amount of bars.